kratos/processes/levelset_convection_process.h

```diff
@@ -774,6 +793,11 @@ class LevelSetConvectionProcess
         mConvectionElementType = GetConvectionElementName(element_type);
         std::string element_register_name = mConvectionElementType + std::to_string(TDim) + "D" + std::to_string(TDim + 1) + "N";
         mpConvectionFactoryElement = &KratosComponents<Element>::Get(element_register_name);
+        mElementRequiresLimiter =  ThisParameters["element_settings"].Has("include_anti_diffusivity_terms") ? ThisParameters["element_settings"]["include_anti_diffusivity_terms"].GetBool() : false;
+        if (mConvectionElementType == "LevelSetConvectionElementSimplexAlgebraicStabilization"){
+            ThisParameters["element_settings"]["requires_distance_gradient"].SetBool(mElementRequiresLimiter);
+        }
```

Comment:
Why changing the parameters?

Reply:
OK. Now I see, you want to base the gradient calculation switch on the antidiffusivity term.

Reply:
Yes. For this specific element, if we need anti-diffusivity, we are obliged to pass the gradient and limiter to the element. So, if mElementRequiresLimiter, then mElementRequiresLevelSetGradient.

Reply:
Thanks for the clarification